internal/pkg/template/templates/workloads/partials/cf/workload-container.yml

```diff
@@ -45,7 +45,7 @@
 {{- end}} {{/* end if eq .WorkloadType "Load Balanced Web Service"*/}}
 {{- if eq .WorkloadType "Backend Service"}}
 {{- if eq .HTTPTargetContainer.Name .WorkloadName}} 
-{{- if and .ServiceConnect .SCFeatureFlag}} # remove when release
+{{- if .SCFeatureFlag}} # remove when release
   PortMappings: !If [ExposePort, [{ContainerPort: !Ref ContainerPort, Name: target}], !Ref "AWS::NoValue"]
```

Comment:
so after we'll just always set `Name`? i assume that's fine?

Reply:
Yeah i think so too.